### DISCUSSION TITLE
Welcome to BlueRetro Discussions!

### DISCUSSION THREAD

**Discussion by darthcloud** [darthcloud/BlueRetro#15] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by julio-benitez** [comment]
hola ! tengo una consulta .
los mandos de sega 8bit-do funcionan con la esp32 blue retro en una ps2?

**Comment by iKero720** [comment]
buenas, tengo controles de PS3, pero dicen wireless, se puede usar este proyecto del ESP32 usandolo en el N64??? gracias por su tiempo...

good, I have PS3 controls, but they say wireless, can I use this ESP32 project using it on the N64??? Thanks for your time...
